app/scripts/lib/xhr.js

```diff
@@ -46,6 +47,40 @@ define([
     },
 
     /**
+     * Low level ajax functionality for OAuth. Sets the `Authorization` header
+     * from `options.accessToken`
+     *
```

Comment:
nit: if adding a jsdoc here, it might be useful to also include `options` if you have time